### PR TITLE
Reduces crabphone volume to being merely loud and reduces the market's hp by 2400

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
## About The Pull Request

Title + also makes it not spawn inside closed walls and makes it not process until it can actually do something inside process()

## Why It's Good For The Game

Fucking christ try playing with sounds on. Also it takes several people several minutes to destroy the fucking thing even if it's immediately found due to high armor, having more HP than bubblegum and being immune or highly resistant to most damage types

## Changelog
:cl: Naksu
balance: The crabphone will no longer deafen everyone who didn't have the foreknowledge that it was coming.
balance: The crabphone has 80% less HP, as befits something that costs 7 TC and is not two bubblegums stacked on top of eachother.
/:cl:

### CODE REVIEW

**Comment by actioninja** [comment]
Fuck, 20 is still loud? Why is this one sound so much louder than so much other stuff?

**Comment by Shadowflame909** [comment]
please buff the amount of money it drains considering the entire station will be after it.

The health goes down pretty fast when 30 people are bashing it with their strongest weapon.

**Comment by tralezab** [comment]
7 tc takes 30 people to intervene

**Comment by Shadowflame909** [comment]
No silly. 7 TC but it tells the entire server of 90 people that they're being robbed and where they're being robbed.

Compare it to the power sink.

**Comment by subject217** [comment]
qustinnus with a poorly balanced meme traitor item? who could have predicted this?!

**Comment by Arianya** [comment]
>The health goes down pretty fast when 30 people are bashing it with their strongest weapon.

This thing exists as a single tile. So at most 8 people could hit it, in a ideal situation. In a more realistic situation, some or most of these tiles will be taken up by walls, tables, or other things which prevent someone hitting it.

So realistically most of the time you have 4 or 5 people hitting it and their best weapon is a toolbox or a lit welder, not a double esword.

**Comment by duckay** [comment]
not accounting for the smart folk who decided to crawl?

**Comment by Arianya** [comment]
Accounting for the minority of the playerbase who would crawl and giving something 3000 hp on top of some pretty nice armour buffs is called "balancing for the far end of the bell curve" and is dumb

**Comment by Shadowflame909** [comment]
By the time the syndicate bomb gets set off every time this is used. It ends up at 2300 from 3000.

This'll make this item being destroyed a breeze.

**Comment by subject217** [comment]
What's the problem, Shadowflame? That's a good thing.

**Comment by JusticeGoat** [comment]
Thank you, i was about to open a issue with how ear shattering loud it is.

**Comment by wesoda25** [comment]
Can we please just remove this already. It’s really annoying to deal with, especially considered only two people ever show up to stop it, meaning they have to beat it with fire extinguishers the whole round, at which point they get blown up or some shit.

**Comment by Shadowflame909** [comment]
@subject217 

It wont be worth 7 tc. Since it'd be reduced to a Loud powersink

**Comment by subject217** [comment]
Yes, and? It shouldn't be in the game to begin with.

**Comment by vuonojenmustaturska** [comment]
i have generously doubled the market's HP in the latest commit, from 300 to 600

File: code/game/objects/items/crab17.dm [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	if(alert(user, "Are you sure you want to crash this market with no survivors?", "Protocol CRAB-17", "Yes", "No") == "Yes")
 		if(dumped || QDELETED(src)) //Prevents fuckers from cheesing alert
 			return FALSE
-		var/turf/targetturf = get_random_station_turf()
+		var/turf/targetturf = get_safe_random_station_turf()
```
Comment (ninjanomnom) [inline code/game/objects/items/crab17.dm]: You should handle the rare case of get_safe_random_station_turf() being null

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**